File: csharp-dotnet2/src/main/CsharpDotNet2/traveltimeplatform/Model/RequestTimeFilterPostcodeSectors.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace traveltimeplatform.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class RequestTimeFilterPostcodeSectors {
    /// <summary>
    /// Gets or Sets DepartureSearches
    /// </summary>
    [DataMember(Name="departure_searches", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "departure_searches")]
    public List<RequestTimeFilterPostcodeSectorsDepartureSearch> DepartureSearches { get; set; }

    /// <summary>
    /// Gets or Sets ArrivalSearches
    /// </summary>
    [DataMember(Name="arrival_searches", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "arrival_searches")]
    public List<RequestTimeFilterPostcodeSectorsArrivalSearch> ArrivalSearches { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class RequestTimeFilterPostcodeSectors {\n");
      sb.Append("  DepartureSearches: ").Append(DepartureSearches).Append("\n");
      sb.Append("  ArrivalSearches: ").Append(ArrivalSearches).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: fsharp-functions/OpenAPI/src/model/RequestTimeFilterPostcodeSectors.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open OpenAPI.Model.RequestTimeFilterPostcodeSectorsArrivalSearch
open OpenAPI.Model.RequestTimeFilterPostcodeSectorsDepartureSearch

module RequestTimeFilterPostcodeSectors = 

  //#region RequestTimeFilterPostcodeSectors

  [<CLIMutable>]
  type RequestTimeFilterPostcodeSectors = {
    [<JsonProperty(PropertyName = "departure_searches")>]
    DepartureSearches : RequestTimeFilterPostcodeSectorsDepartureSearch[];
    [<JsonProperty(PropertyName = "arrival_searches")>]
    ArrivalSearches : RequestTimeFilterPostcodeSectorsArrivalSearch[];
  }
  
  //#endregion
  